src/libDirectoryService/DirectoryService.h

```diff
@@ -616,7 +629,8 @@ class DirectoryService : public Executable {
           coinbase_rewardees);
 
   /// Implements the Execute function inherited from Executable.
-  bool Execute(const bytes& message, unsigned int offset, const Peer& from);
+  bool Execute(const bytes& message, unsigned int offset, const Peer& from,
+               const unsigned char& startByte = START_BYTE_NORMAL);
```

Comment:
I can't find a call to `Execute` that does not have the `startByte` parameter. Should we just remove the default `= START_BYTE_NORMAL` value?

Reply:
yeah.done